### PR TITLE
Main update

### PR DESCRIPTION
**Pull request by edichoska**

### CODE REVIEW

File: components/module-components/projectsSection/projectData.ts
Comment (RevanchistX) [inline components/module-components/projectsSection/projectData.ts:1]: Replace `lorem ipsum` for something more substantial.

**Review by RevanchistX — CHANGES_REQUESTED** [review]
LGTM, just change the dummy data

File: components/module-components/projectsSection/projectData.ts
Comment (edichoska) [reply components/module-components/projectsSection/projectData.ts:1]: @RevanchistX someone else from the team will update the remaining lorem ipsum text once we come up with descriptions. This is so we update main

Comment (davorminchorov) [reply components/module-components/projectsSection/projectData.ts:1]: I would suggest that we update the text first before releasing the new design.

**Review by RevanchistX — APPROVED** [review]

**Pull request merged**